cips/cip-prevote.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Coordinated prevote times
+description: Scheduled prevote times for consistent blocks
+author: Callum Waters (@cmwaters)
+discussions-to: URL
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-07-12
+---
+
+## Abstract
+
+Block propagation speed is dependent on block size, thus larger blocks have later block times. It is desirable that regardless of block size that blocks are finalized with a consistent cadence. This CIP proposes a solution to this problem, namely, correct processes that receive a valid proposal withthold their prevote until the `ProposalTimeout` has elapsed.
+
+## Motivation
+
+The current network has a cap of 2MB. We have observed in testnets that larger blocks (~8MB) shift the block time by up to 7 seconds (18 . seconds in total with the default 11 second `TimeoutCommit`). This range of 1 - 7 seconds for blocks 0-8MB in size is too inconsistent for users. We want to provide block finality with a consistent cadence.
```

Comment:
[nit] errant period.

nit: does it make sense to rephrase the range in terms of the observed block interval because that is what users perceive: "This range of 1 - 7 seconds" => "This range of 11 - 18 seconds"

```suggestion
The current network has a cap of 2MB. We have observed in testnets that larger blocks (~8MB) shift the block time by up to 7 seconds (18 seconds in total with the default 11 second `TimeoutCommit`). This range of 11 - 18 seconds for blocks 0-8MB in size is too inconsistent for users. We want to provide block finality with a consistent cadence.
```